Framework/CurveFitting/src/Algorithms/ConvolveWorkspaces.cpp

```diff
@@ -31,9 +31,9 @@ using namespace Functions;
 
 void ConvolveWorkspaces::init() {
   declareProperty(std::make_unique<WorkspaceProperty<Workspace2D>>("Workspace1", "", Direction::Input),
```

Comment:
Is it worth having a V2 version of this algorithm with better named properties, given that we're changing some of the internal behaviour anyway?

Reply:
Thanks for reviewing. My feeling is that leaving it as v1 is probably OK. Although I've changed the internal behaviour I think this is pretty clearly a bug fix rather than a change in behaviour from one sensible approach to another. I guess I could rename the properties to hammer home the fact that they're not switchable\equivalent but hopefully my documentation change is sufficient.

Reply:
My gut feeling is that if we're already changing behaviour it would be better to have the workspaces renamed. `Workspace1` and `Workspace2` aren't really useful at all to the user

Reply:
I think the new algorithm versions are really for functional improvements rather than bug fixes. What I'm aiming to do here is just fix the bug. In theory I could fix the bug in v1 and create a v2 with better parameter names but I'm choosing not to do that because I don't have time.

Reply:
That's fair 👍